tests/PregTest.php

```diff
@@ -86,7 +84,7 @@ public function testPatternValidation(string $pattern, ?bool $expected = null, ?
 
             if (null !== $expectedMessage) {
                 ++$i;
-                $this->expectExceptionMessage($expectedMessage);
+                $this->expectExceptionMessageMatches($expectedMessage);
```

Comment:
why the change?

Reply:
Removed line:

> $nullByteMessage = \PHP_VERSION_ID >= 8_02_00 ? 'NUL is not a valid modifier' : 'Null byte in regex';

We don't have to expect the message based on the PHP version, we can simply check if the message matches any of these two.